DESCRIPTION

```diff
@@ -24,11 +24,13 @@ Imports:
     fs (>= 1.2.6),
     getPass (>= 0.2-2),
     glue (>= 1.3.1),
+    here (>= 0.1),
     htmltools (>= 0.3.6),
     httr (>= 1.4.0),
     jsonlite (>= 1.6),
     magrittr (>= 1.5),
     processx (>= 3.3.0),
+    rmarkdown,
```

Comment:
FYI @rich-iannone I made this an Imports so that rmarkdown revdepcheck will run our tests. This will give us the earliest warning if something goes wrong with the sidecar path hack.

Reply:
Thanks!